Resources/views/settings_overtime_history.html.twig

```diff
@@ -35,7 +35,7 @@
                         <td>
                             {% set deleteLinkPath = path('delete_overtime_history', {'entryId': entry.id }) %}
                             <button type="button" class="btn btn-sm btn-warning closeModal" 
-                                    onclick="$('#deniedLink').attr('href', '{{ deleteLinkPath }}');">
+                                    onclick="window.location.href='{{deleteLinkPath}}'">
```

Comment:
I don't know if this change is related with jQuery that was removed from Kimai.

If yes, your change is a good approach, but have a lot of places using jQuery.

I made this workaround that maybe fixed all problems related as jQuery
https://github.com/KatjaGlassConsulting/ApprovalBundle/commit/bee83f97e708fdf2a809c5233882a17fe5edb68c

I know that isn't the best because is a request to an external package, but was the most simple and fast solution that I found.

Reply:
Either works, but I agree that it should be standardized throughout. I went with this method as it was already used in a few other places throughout the codebase and didn't require any external dependencies.